src/styles/main.scss

```
@@ -0,0 +1,61 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  height: 100vh;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  width: 726px;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+
+  &__day {
+    border: 1px solid black;
+    height: 100px;
+    width: 100px;
+    background-color: #eee;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__day:hover {
+    cursor: pointer;
+    background-color: #ffbfcb;
+    transform: translateY(-20px);
+    transition-duration: 0.5s;
+  }
+
+  @for $num from 1 through 31 {
+    &__day:nth-child(#{$num})::before {
+      content: '#{$num}';
+    }
+  }
+}
+
+$day: (mon 0, tue 101, wed 202, thu 303, fri 404, sat 505, sun 606);
```

Move variable to the separate file. The indentation must be counted inside the loop
```suggestion
$day: (mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6);
```